minishift/minishift
# Issue #2696 Add maximum time limit to start and stop of Che workspaces

Fixes #2696 


Steps to test the pull request

  1. make integration GODOG_OPTS=-tags=addon-che

You can lower maximum time in addon-che.feature to `1s` or `1ms` to see the behaviour when the scenario timeouts.
